jenkins/agent-base/Dockerfile.ubi8

```diff
@@ -39,10 +40,22 @@ RUN cd /tmp \
 
 # Install Tailor.
 RUN cd /tmp \
-	&& curl -LO https://github.com/opendevstack/tailor/releases/download/v${TAILOR_VERSION}/tailor-linux-amd64 \
-	&& mv tailor-linux-amd64 /usr/local/bin/tailor \
-	&& chmod a+x /usr/local/bin/tailor \
-	&& tailor version
+    && curl -LO https://github.com/opendevstack/tailor/releases/download/v${TAILOR_VERSION}/tailor-linux-amd64 \
+    && mv tailor-linux-amd64 /usr/local/bin/tailor \
+    && chmod a+x /usr/local/bin/tailor \
+    && tailor version
+
+# Install Helm.
+RUN cd /tmp \
+    && mkdir -p /tmp/helm \
+    && curl -LO https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \
+    && tar -zxvf helm-v${HELM_VERSION}-linux-amd64.tar.gz -C /tmp/helm \
+    && mv /tmp/helm/linux-amd64/helm /usr/local/bin/helm \
+    && chmod a+x /usr/local/bin/helm \
+    && helm version \
+    && helm plugin install https://github.com/databus23/helm-diff --version v3.1.3 \
+    && helm plugin install https://github.com/jkroepke/helm-secrets --version v3.3.0 \
```

Comment:
should we make these plugins versions also ENVs?

Reply:
Why not, added in 5d93086.